Source/santa-driver/SantaDecisionManager.cc

```diff
@@ -208,6 +219,83 @@ kern_return_t SantaDecisionManager::StopListener() {
   return kIOReturnSuccess;
 }
 
+# pragma mark Monitoring PIDs
+
+// Arguments that are passed to pid_monitor thread.
+typedef struct {
+  pid_t pid;                  // process to monitor
+  SantaDecisionManager *sdm;  // reference to SantaDecisionManager
```

Comment:
Can you add a TODO somewhere appropriate to split this out of SDM at some point? I'd like to make SDM smaller and I think this can almost entirely live on its own.

Reply:
Done.